tsconfig.json

```diff
@@ -1,11 +1,10 @@
 {
   "compilerOptions": {
     "outDir": "lib",
-    "target": "es5",
+    "target": "es6",
```

Comment:
Any desire to go further than this? Bigint was added in ES2020 but I don't know the downside of using a fairly new compiler target

Reply:
i think going to es6 is fine for now. we can do es2020 with ethers v6 + bigint upgrade